way_point_gap: 0.5 # meter
number_Of_Lookahead_Wp: 5
wheel_base: 0.87 # meter